TrkHitReco/src/TNTClusterer.cc

```diff
@@ -6,29 +6,42 @@
 namespace mu2e
 {
   TNTClusterer::TNTClusterer(const Config& config) :
-    dhit_       (config.hitDistance()),
-    dseed_      (config.seedDistance()),
-    dd_         (config.clusterDiameter()),
-    dt_         (config.clusterTime()),
-    maxHitdt_   (config.maxHitTimeDiff()),
-    maxDistSum_ (config.maxSumDistance()),
-    maxNiter_   (config.maxCluIterations()),
-    useMedian_  (config.medianCentroid()),
-    comboInit_  (config.comboInit()),
-    bkgmask_    (config.bkgmsk()),
-    sigmask_    (config.sigmsk()),
-    testflag_   (config.testflag()),
-    diag_       (config.diag())
+    dd_               (config.clusterDiameter()),
+    dt_               (config.clusterTime()),
+    minClusterHits_   (config.minClusterHits()),
+    maxHitdt_         (config.maxHitTimeDiff()),
+    maxDistSum_       (config.maxSumDistance()),
+    maxNiter_         (config.maxCluIterations()),
+    useMedian_        (config.medianCentroid()),
+    comboInit_        (config.comboInit()),
+    bkgmask_          (config.bkgmsk()),
+    sigmask_          (config.sigmsk()),
+    testflag_         (config.testflag()),
+    diag_             (config.diag()),
+    distMethod_       (config.distMethod())
   {
-     float minerr (config.minHitError());
-     float maxdist(config.maxDistance());
-     float trms   (config.timeRMS());
-
-     tbin_     =1.0;
-     dd2_      = dd_*dd_;
-     maxwt_    = 1.0f/minerr;
-     md2_      = maxdist*maxdist;
-     trms2inv_ = 1.0f/trms/trms;
+    float minerr (config.minHitError());
+    float maxdist(config.maxDistance());
+    float trms   (config.timeRMS());
+
+    tbin_     =1.0;
+    dd2_      = dd_*dd_;
+    maxwt_    = 1.0f/minerr;
+    md2_      = maxdist*maxdist;
+    trms2inv_ = 1.0f/trms/trms;
+
+    switch(distMethod_) {
+      case TNTClusterer::useDistance:
+        dhit_ = 5.;//mm
```

Comment:
I am confused why these hard-coded numbers are needed.  Shouldn't they come from the config object at initialization?

Reply:
Misunderstanding on my part, I'll change it back